example/server-compress.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const path = require('path')
+const fastify = require('fastify')({ logger: { level: 'trace' } })
+
+fastify
+  // compress everything
+  .register(require('fastify-compress'), { global: true, threshold: 0 })
```

Comment:
You don't need `global: true` it's the [default](https://github.com/fastify/fastify-compress#global-hook) ;)